common/test/com/commercetools/sunrise/common/ctp/MetricHttpClientTest.java

```diff
@@ -1,65 +1,67 @@
-package io.sphere.sdk.play.metrics;
+package com.commercetools.sunrise.common.ctp;
 
 import io.sphere.sdk.http.HttpClient;
 import io.sphere.sdk.http.HttpMethod;
 import io.sphere.sdk.http.HttpRequest;
 import io.sphere.sdk.http.HttpResponse;
 import org.junit.Test;
 import play.mvc.Http;
+import org.slf4j.LoggerFactory;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
 
 import java.util.LinkedList;
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
-import java.util.function.Consumer;
+import java.util.function.BiConsumer;
 
-import static io.sphere.sdk.play.metrics.MetricAction.KEY;
 import static java.util.Collections.emptyMap;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class MetricHttpClientTest {
     private static final HttpRequest SOME_REQUEST = HttpRequest.of(HttpMethod.GET, "somewhere");
     private static final HttpResponse SOME_RESPONSE = HttpResponse.of(200);
 
-    @Test
-    public void executesRequestWithNoContext() throws Exception {
-        executeWithoutContext(SOME_REQUEST, response -> assertThat(response).isEqualTo(SOME_RESPONSE));
-    }
+// TODO Do we still need this?
```

Comment:
I will remove it, with the new architecture it is always the case that it has the HTTP.Context.